### PR TITLE
Added Webcord, office apps, and some scientific computing apps by @gabepoel

### PR DESCRIPTION
**Pull request by iaacornus**
Added Webcord, office apps, and some scientific computing apps by @gabepoel:
- [X] webcord
- [X] matematica
- [X] okular
- [X] office apps and mimes
- [X] octave
- [X] maple
- [X] svgs and kde mimetypes
- [X] google calendar
- [X] google contacts
- [X] gdrive
- [X] maps
- [X] gphotos
- [X] microsoft excel
- [X] microsoft word
- [X] microsoft powerpoint
- [X] microsoft teams
- [X] outlook
- [X] onenote
- [X] oasis dbtemplate
- [X] ipynb mime #13 
- [X] oasis db
- [X] oasis empty
- [X] empty template
- [X] office spreadsheet mimes
- [X] octave mimes
- [X] oasis drawing template
- [X] oasis drawing
- [X] oasis formula
- [X] oasis formula template
- [X] oasis masterdoc
- [X] oasis masterdoc template
- [X] oasis presentation
- [X] oasis presentation template
- [X] oasis text
- [X] oasis text template
- [X] oasis spreadsheet
- [X] oasis spreadsheet template
- [X] wolfram cdf
- [ ] testing
- [ ] test symlinks

### CODE REVIEW

File: apps/scalable/mathematica.svg
Comment (iaacornus) [inline apps/scalable/mathematica.svg:1]: rendering error

File: apps/scalable/teams.svg
Comment (iaacornus) [inline apps/scalable/teams.svg:1]: also erroneous

File: mimes/scalable/application-x-kdenlive.svg
Comment (iaacornus) [inline mimes/scalable/application-x-kdenlive.svg:1]: inconsistent with style

File: mimes/scalable/application-x-krita.svg~a8fa95e90f21de042cdc3e4a2a44ef437c81eabe
Comment (iaacornus) [inline mimes/scalable/application-x-krita.svg~a8fa95e90f21de042cdc3e4a2a44ef437c81eabe:1]: file cant render

**Comment by yioannides** [comment]
Make sure every new icon coming in strictly adheres to GNOME's HIG, as well as to refer to the reasons existing PRs weren't merged to begin with. Also, +1 for `neowaita` 👍️, but I would also rename the organization's name, cause `MoreWaita-Dev` sounds confusion, as well as a project tied to the original author

**Comment by iaacornus** [comment]
> Make sure every new icon coming in strictly adheres to GNOME's HIG, as well as to refer to the reasons existing PRs weren't merged to begin with. Also, +1 for `neowaita` 👍️, but I would also rename the organization's name, cause `MoreWaita-Dev` sounds confusing, as well as a project tied to the original author

yes. Icons not adhering to gnome's HIG and design philosophy will be revamped or revised before merging the pr, if not, it will be excluded. Some of papirus' icon are adherent hence I'm thinking of absorbing it to create a fusion of papirus and adwaita but consistent with gnome's styling. Thank you for your input for the fork's name, I'll see what's the best name once the final philosophy and path of the fork is settled.

**Pull request merged**